definitions/npm/react-redux_v4.x.x/flow_>=v0.24.x/react-redux_v4.x.x.js

```diff
@@ -0,0 +1,33 @@
+type ReactReduxConnectAll = <D, P, S, C: React$Component<D, P, S>, SP, DP, Dispatch: Function>(
+  mapStateToProps: (state: Object) => SP,
+  mapDispatchToProps: (dispatch: Dispatch) => DP,
```

Comment:
I also made a pull request to `react-redux` with the types so it's easier to get the types. Turns out, there's no good way so far to do type definitions for case where `mapDispatchToProps` is an object.

I'm thinking of possible work arounds.